hackathon_site/event/models.py

```diff
@@ -27,6 +28,16 @@ class Profile(models.Model):
     team = models.ForeignKey(
         Team, related_name="profiles", on_delete=models.CASCADE, null=False
     )
+    phone_number = models.CharField(
+        max_length=20,
+        null=False,
+        validators=[
```

Comment:
the validators argument is required in the registration model because we use a django form to create an application. Here, there will be no form so you can remove it.

Reply:
removed